README.md

```diff
@@ -94,6 +94,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 15 Advanced (Early 2022)          |  1532:028A  |
 | Razer Blade 17 (2022)                         |  1532:028B  |
 | Razer Blade 14 (2022)                         |  1532:028C  |
+| Razer BlackWidow V4                           |  1532:0287  |
```

Comment:
Keep this list sorted by USB PID, so move above 028A

Reply:
Ok, fixed.